com.unity.multiplayer.mlapi/Runtime/Core/NetworkedBehaviour.cs

```diff
@@ -80,7 +80,7 @@ public abstract partial class NetworkedBehaviour : MonoBehaviour
         /// Gets if we are executing as Host, I.E Server and Client
         /// </summary>
         protected bool IsHost => IsRunning && NetworkingManager.Singleton.IsHost;
-        private bool IsRunning => NetworkingManager.Singleton != null && NetworkingManager.Singleton.IsListening;
+        private static bool IsRunning => NetworkingManager.Singleton != null && NetworkingManager.Singleton.IsListening;
```

Comment:
same here, we probably don't want to turn this into a static.

Reply:
This unfortunately also needs to be static to support IsServer being a static.